termite.cc

```diff
@@ -44,6 +44,21 @@
 
 #include "url_regex.hh"
 
+typedef struct {
+  const char *pattern;
+} TerminalRegexPattern;
+
+static const TerminalRegexPattern url_regex_patterns[] = {
+  REGEX_URL_AS_IS,
+  REGEX_URL_HTTP,
+  REGEX_URL_FILE,
+  REGEX_URL_VOIP,
+  REGEX_EMAIL,
+  REGEX_NEWS_MAN,
+};
+
+static guint n_url_regexes;
```

Comment:
This is not needed, we can use `G_N_ELEMENTS()` in the loop below.

```suggestion
```

Reply:
yes, thanks